### DISCUSSION TITLE
WebhookCall is losing information

### DISCUSSION THREAD

**Discussion by xmas** [spatie/laravel-webhook-client#171] [Q&A]
Hi, I was using this package to receive webhook calls from Mailgun. When the request comes in via WebhookCall, the resulting `payload` is missing the attachments that were attributes on the original request.

Ideally this data would not be lost.

The missing values were:

```
  'attachment-1' => 
  \Illuminate\Http\UploadedFile::__set_state(array(
     'test' => false,
     'originalName' => 'crabby.gif',
     'mimeType' => 'image/gif',
     'error' => 0,
     'hashName' => NULL,
  )),
  'attachment-2' => 
  \Illuminate\Http\UploadedFile::__set_state(array(
     'test' => false,
     'originalName' => 'attached_файл.txt',
     'mimeType' => 'text/plain',
     'error' => 0,
     'hashName' => NULL,
  )),
```

**Comment by freekmurze** [comment]
That's strange, we do not try to manipulate the payload. 

Could you add a failing test to prove that something goes wrong in our package?
